packages/framework/src/Model/Mail/Mailer.php

```diff
@@ -5,20 +5,54 @@
 use Swift_Attachment;
 use Swift_Mailer;
 use Swift_Message;
+use Symfony\Component\HttpKernel\KernelInterface;
 
 class Mailer
 {
+    /**
+     * @var \Symfony\Component\HttpKernel\KernelInterface
+     */
+    protected $kernel;
+
     /**
      * @var \Swift_Mailer
      */
     protected $swiftMailer;
 
     /**
      * @param \Swift_Mailer $swiftMailer
+     * @param \Symfony\Component\HttpKernel\KernelInterface $kernel
      */
-    public function __construct(Swift_Mailer $swiftMailer)
+    public function __construct(Swift_Mailer $swiftMailer, KernelInterface $kernel)
```

Comment:
I don't think that you should get the whole KernelInterface. Can't you pass the mailers as parameter?

Reply:
+ missing upgrade notes

Reply:
hmm this is good idea, how could it look like to pass mailers as parameters?

Reply:
upgrade notes about ? 
`fix constructor of FrameworkBundle/Model/Mail/Mailer if you extended it somehow.` ???